docs/guide/major-changes.md

```diff
@@ -162,7 +162,7 @@ On the other hand, `@apostrophecms/page:beforeSend` will not be listened for as
 
 ### Queries
 
-In modules for piece and page types, the `queries` section replaces the Apostrophe 2.x "cursors" feature. When making a database query with Apostrophe, you can chain together the query "builders" declared in the `builders` subsection, then finish the job with the query "methods" declared in the `builders` subsection. Most of the time you'll just need those you inherit from `@apostrophecms/doc-type`, such as `and`, `project`, and `toArray`.
+In modules for piece and page types, the `queries` section replaces the Apostrophe 2.x "cursors" feature. When making a database query with Apostrophe, you can chain together the query "builders" declared in the `builders` subsection, then finish the job with the query "methods" declared in the `methods` subsection. Most of the time you'll just need those you inherit from `@apostrophecms/doc-type`, such as `and`, `project`, and `toArray`.
```

Comment:
... As well as the query builders that are added automatically for most non-area fields in your schema. (In the "most of the time you'll just need" section)

Reply:
This was simply fixing a typo. We'll be totally rewriting this page later.